### PR TITLE
recipes: add initial Bugzilla service

### PR DESCRIPTION
**Pull request by xgqt**
Closes: https://github.com/ferdium/ferdium-recipes/issues/150
Signed-off-by: Maciej Barć <xgqt@gentoo.org>

### CODE REVIEW

**Comment by xgqt** [comment]
@vraravam ping
Could you check it one out? Thanks in advance. :D

File: recipes/bugzilla/package.json
Comment (vraravam) [inline recipes/bugzilla/package.json:8]: Please provide a default url for bugzilla

**Review by vraravam — CHANGES_REQUESTED** [review]

**Review by vraravam — APPROVED** [review]
The empty webview will mean that you don't get notifications/unread badges, etc. I hope that's fine. Approving with this caveat

**Comment by vraravam** [comment]
@xgqt - please see the reason for the broken build. Please fix before this can be merged.

**Comment by xgqt** [comment]
> @xgqt - please see the reason for the broken build. Please fix before this can be merged.

Sure, I will take care of reshaping the svg file.

**Review by vraravam — APPROVED** [review]
Accepting the base shell. Need to fix issues as they arise based on unread counter badges, etc

**Comment by vraravam** [comment]
@all-contributors please add @xgqt for code

**Pull request merged**
